core/src/uix/compiler/debug.cljs

```diff
@@ -29,7 +29,10 @@
         (if (string? name)
           name))))
 
-(defn with-name [^js f]
-  (when-let [component-name (effective-component-name f)]
+(defn with-name
+  ([^js f ns-str sym-str]
+   (set! (.-displayName f) (str ns-str "/" sym-str)))
+  ([^js f]
+   (when-let [component-name (effective-component-name f)]
     (when-some [display-name (format-display-name component-name)]
-      (set! (.-displayName f) display-name))))
+      (set! (.-displayName f) display-name)))))
```

Comment:
Maybe we can avoid the nested `let` bindings with `some->>`?

```cljs
(some->> f
  effective-component-name
  format-display-name
  (set! (.displayName f)))

Reply:
I'd prefer to keep it more explicit, also having a side effectful operation as in a threading macro could be confusing imo